src/content/docs/tutorial-create-alerts/manage-alert-quality.mdx

```diff
@@ -0,0 +1,182 @@
+
+### Incident volume [#volume]
+
+You should treat incidents (with or without alerts) like a queue of tasks. Just like a queue, the number of alerts should always be as close to zero as possible. Each incident should trigger an investigatory or corrective action to resolve the condition. If an alert doesn't result in some sort of action, then you should question the value of the alert condition.
+
+In particular, if you see specific incidents that are frequently triggered, then you should question whether you're in a constant state of meaningful impact or if you simply have a large volume of noise? The incident volume KPIs help you answer those questions and measure progress towards a healthy state of high quality alerting.
+
+<CollapserGroup>
+  <Collapser
+    id="kpi-incident-count"
+    title="Incident count KPI"
+  >
+This is the number of incidents generated over a period of time. Typically you should compare the current and previous weeks.
+
+**Goal:** Reduce the number of low value / nuisance incidents.
+
+**Best practices:**
+* Ensure condition settings are intended to detect real business impact.
+* Ensure condition settings are detecting abnormal behavior.
+* Communicate that the incident details `Acknowledge` feature helps measure meaningful and actionable alerts. See [percentage incident acknowledge KPI](#kpi-user-engagement).
+* Report AQM KPIs to all stakeholders.
+
+```sql
+FROM NrAiIncident SELECT count(*) AS 'Incident Count' WHERE event = 'open' AND priority = 'critical' SINCE 1 WEEK AGO COMPARE WITH 1 WEEK AGO
+```
+  </Collapser>
+
+  <Collapser
+    id="kpi-incident-duration"
+    title="Accumulated incident duration KPI"
+  >
+This is the total sum of minutes that all incidents have accumulated over a period of time. Typically you should compare the current and previous weeks.
+
+**Goal:** Reduce the total accumulated minutes of incidents.
+
+**Best practices:**
+* Avoid manually closing incidents that don't motivate some sort of investigatory action. Manual closure will skew the real duration of incident length.
+* Remove alerts that don't result in any remediation actions from the recipients.
+* Improve **percent investigated** and **mean-time-to-investigate** KPIs by communicating their importance in improving detection and response times.
+* Report AQM KPIs to all stakeholders.
+
+```sql
+FROM NrAiIncident SELECT sum(durationSeconds)/60 AS 'Incident Minutes' WHERE event = 'close' AND priority = 'critical' SINCE 1 WEEK AGO COMPARE WITH 1 WEEK AGO
+```
+  </Collapser>
+
+  <Collapser
+    id="kpi-mttc"
+    title="Mean time to close (MTTC) KPI"
+  >
+This is simply the average duration of incidents within the period of time measured. You want this number to be as low as possible.
```

Comment:
I love the very simple, direct language here! This is how you would communicate it verbally.